### PR TITLE
[MOO-1090] Token Store's file storing functions updated.

### PR DESCRIPTION
**Pull request by stelselim**
**Fixes**
- resolveLocalFileSystemURL function is introduced in TokenStore.
- New Android tag added. 

<br/> 

**References**
- [Github Issue](https://github.com/apache/cordova-plugin-file/issues/408)

### CODE REVIEW

**Review by omnajjar — APPROVED** [review]

File: src/www/scripts/file-store.js [hunk 1]
```diff
@@ -3,11 +3,10 @@
 export default (function () {
     var remove = function (namespace, key) {
         return new Promise(function (resolve, reject) {
-            window.requestFileSystem(
-                LocalFileSystem.PERSISTENT,
-                0,
+            window.resolveLocalFileSystemURL(
+                cordova.file.dataDirectory,
                 function (fs) {
-                    fs.root.getFile(
```
Comment (vadymv-mendix) [inline src/www/scripts/file-store.js:10]: I wonder why did we even write to the root directory in the first place, so this solution looks great :)

**Review by vadymv-mendix — APPROVED** [review]

File: src/config.xml.mustache [hunk 2]
```diff
@@ -193,6 +193,10 @@
                 <application android:exported="true" />
             </edit-config>
 
+            <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
+                <application android:usesCleartextTraffic="true" android:requestLegacyExternalStorage="true"/>
```
Comment (mlaponder) [inline src/config.xml.mustache:197]: Is the `usesClearTraffic` needed? Iiuc this is not needed when when are communicating over https (which is a requirement for the cookies to persist).

**Review by mlaponder — COMMENTED** [review]
I think it would be nice to document in the readme the requirement of https and the Mendix runtime setting which is needed for the hybridapp.

File: src/www/scripts/file-store.js (hunk 1 above)
Comment (stelselim) [reply src/www/scripts/file-store.js:10]: I am not sure about it, this function was implemented years ago. I guess it is related to **requestFileSystem function**.

**Pull request merged**